src/audio/heap.c

```diff
@@ -421,6 +421,7 @@ void *alloc_bank_or_seq(struct SoundMultiPool *arg0, s32 arg1, s32 size, s32 arg
             break;
 
         case 2:
+        default:
```

Comment:
you're sure this should be a default?

Reply:
Assuming the error here is arg0 being uninitialized, I would just set it to NULL at the beginning of the function and remove the default.

Reply:
done